### PR TITLE
fixes-nov-29

### PR DESCRIPTION
**Pull request by nvmbrasserie**

### CODE REVIEW

File: datasets/fr/tresor/crawler.py [hunk 1]
```diff
@@ -8,6 +8,11 @@
 from zavod import Context, Entity
 from zavod import helpers as h
 
+HEADERS = {
+    "Accept": "application/json",
+    "User-Agent": "custom-agent",
```
Comment (jbothma) [inline datasets/fr/tresor/crawler.py:13]: It's nice to include opensanctions.org in the user agent if we can make it work with that text in there. That way if we hurt them they can get in touch with us easily.

File: datasets/nz/russia_sanctions/nz_russia_sanctions.yml [hunk 2]
```diff
@@ -42,6 +42,7 @@ lookups:
       - match:
           - unknown
           - 08/15/1977
+          - 45625
```
Comment (jbothma) [inline datasets/nz/russia_sanctions/nz_russia_sanctions.yml:45]: fun fact: this is an excel date

```
 h.convert_excel_date(45625)
'2024-11-29T00:00:00'
```

File: datasets/fr/tresor/crawler.py (hunk 1 above)
Comment (nvmbrasserie) [reply datasets/fr/tresor/crawler.py:13]: It doesn't work unfortunately in this case

File: datasets/nz/russia_sanctions/nz_russia_sanctions.yml (hunk 2 above)
Comment (nvmbrasserie) [reply datasets/nz/russia_sanctions/nz_russia_sanctions.yml]: I suspected it, but couldn't find a way to verify it :)

**Pull request merged**